pymeasure/instruments/attocube/anc300.py

```diff
@@ -221,9 +221,9 @@ class ANC300Controller(Instrument):
 
     def __init__(self, host, axisnames, passwd, query_delay=0.05, **kwargs):
         kwargs['query_delay'] = query_delay
+        kwargs.setdefault('name', "attocube ANC300 Piezo Controller")
```

Comment:
Same as in the SerialAdapter example above, the `AttocubeConsoleAdapter` does not accept  a `name` kwarg afaik.

Reply:
Ok.